### PR TITLE
add(cassandra): AE updates for CCR docs

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-837

1. PRO label added to the doc titles
2. Pro Platform requirement added to PREREQUISITES
3. Editorial improvements

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/cassandra/howto/disable-cross-cluster-replication.md [hunk 1]
```diff
@@ -61,15 +57,15 @@ your services for CCR purposes.
 
 1.  Log in to [Aiven Console](https://console.aiven.io/).
 
-2.  From the **Services** page, select an Aiven for Apache Cassandra
+1.  From the **Services** page, select an Aiven for Apache Cassandra
     service on which you'd like to enable CCR.
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.md:61]: This should say "to disable CCR" as this section is about disabling CCR by deleting services

File: docs/products/cassandra/howto/disable-cross-cluster-replication.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
 ---
 title: Disable cross-cluster replication in Aiven for Apache Cassandra®
+pro: true
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.md:3]: Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise

File: docs/products/cassandra/howto/enable-cross-cluster-replication.md [hunk 3]
```diff
@@ -1,76 +1,63 @@
 ---
 title: Enable cross-cluster replication in Aiven for Apache Cassandra®
+pro: true
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/howto/enable-cross-cluster-replication.md:3]: Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise

File: docs/products/cassandra/howto/list-cross-cluster-replication.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 ---
 title: Cross-cluster replication
+pro: true
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/howto/list-cross-cluster-replication.md:3]: Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise

File: docs/products/cassandra/howto/manage-cross-cluster-replication.md [hunk 5]
```diff
@@ -1,33 +1,21 @@
 ---
 title: Manage cross-cluster replication in Aiven for Apache Cassandra®
+pro: true
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/howto/manage-cross-cluster-replication.md:3]: Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise

File: docs/products/cassandra/concepts/cross-cluster-replication.md [hunk 6]
```diff
@@ -1,9 +1,10 @@
 ---
 title: Cross-cluster replication with Aiven for Apache Cassandra®
+pro: true
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/concepts/cross-cluster-replication.md:3]: Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise

File: docs/products/cassandra/howto/enable-cross-cluster-replication.md [hunk 7]
```diff
@@ -1,76 +1,63 @@
...
+    1.  Enter names for your services.
 
 :::note[Datacenters names]
 Notice that the service names you enter are automatically copied as the
 names for datacenters for the services. The datacenters names can be
 required when executing statements on your databases.
 :::
 
-4.  Verify your choices in the **Services summary** card on the right
+1.  Verify your choices in the **Services summary** card on the right
```
Comment (ArthurAnsquer) [inline docs/products/cassandra/howto/enable-cross-cluster-replication.md:60]: I think formatting got strange and this should be "4." but got set automatically to "1."

**Review by ArthurAnsquer — CHANGES_REQUESTED** [review]

**Review by ArthurAnsquer — APPROVED** [review]

**Review by ArthurFlag — DISMISSED** [review]
Let's rebase on main please, the review contains incorrect changes (I think) :)

File: docs/products/cassandra/howto/manage-cross-cluster-replication.md [hunk 8]
```diff
@@ -101,7 +91,7 @@ adding storage space for CCR-enabled services.
 
 1. Select **Save changes**.
 
-You've added extra disk storage space for your CCR-enabled service and
+You\'ve added extra disk storage space for your CCR-enabled service and
```
Comment (ArthurFlag) [inline docs/products/cassandra/howto/manage-cross-cluster-replication.md:94]:
```suggestion
You've added extra disk storage space for your CCR-enabled service and
```

File: docs/products/cassandra/howto/manage-cross-cluster-replication.md [hunk 9]
```diff
@@ -223,31 +215,35 @@ parameter or object to define the consistency level on your software
...
 `SimpleStatement` object.
 
 ```bash
 session.execute(SimpleStatement("LIST ROLES", consistency_level=ConsistencyLevel.ALL))
 ```
+
 :::
 
-You've set up the consistency level for your service. Now operations on
+You\'ve set up the consistency level for your service. Now operations on
```
Comment (ArthurFlag) [inline docs/products/cassandra/howto/manage-cross-cluster-replication.md:228]:
```suggestion
You've set up the consistency level for your service. Now operations on
```

**Pull request closed without merging**